tests/scripts/create-dev-cluster.sh

```diff
@@ -54,21 +57,27 @@ show_info() {
     fi
     echo "==========================="
     echo " "
-    echo " *** To start using your rook cluster please set the following alias: "
+    echo " *** To start using your rook cluster please set the following env: "
+    echo " "
+    echo "   > eval \$($MINIKUBE docker-env)"
+    echo "   > alias kubectl=\"$KUBECTL"\"
+    echo " "
+    echo " *** To access the new cluster with k9s: "
+    echo " "
+    echo "   > k9s --context $ROOK_PROFILE_NAME"
```

Comment:
we should also update this in the doc section

Reply:
> we should also update this in the doc section

@parth-gr AFAIR Good idea. What I will do is added some more information of the possibility of having multiple clusters and how to run them.